docket/application.py

```diff
@@ -48,38 +52,39 @@ def _call_(self, *args, **kwargs):
 
         return celery
 
-    def manager(self):
-        return Manager(self.flask_app)
+#    def manager(self):
+#        return Manager(self.flask_app)
 
     def _set_blueprints(self):
         from api import api_bp
         self.flask_app.register_blueprint(api_bp)
 
     def _configure_app(self, env):
-        _conf_path = env.get('APP_CONFIG', 'devel')
-        self.flask_app.logger.info("Loading config: %s" % _conf_path)
-        _config = {}
-        if _conf_path:
-            self.flask_app.config['APP_CONFIG'] = os.path.join(
-                '/etc/docket/', ("%s.yaml" % _conf_path))
+        conf = env.get('DOCKET_CONF') or env.get('APP_CONF')
+        config = Config.load(conf, flask_app=self.flask_app)
+
+        celery_url = env.get('CELERY_URL', config['CELERY_URL'])
 
-            with open(self.flask_app.config['APP_CONFIG']) as f:
-                _config = yaml.load(f.read())
-            self.flask_app.logger.debug("Loaded YAML:\n%s" % _config)
+        if not celery_url:
+            Config.logger.critical("Unable to locate 'CELERY_URL', it configures the broker.")
+            raise Exception("Invalid configuration / Environment: "
+                            "Unable to locate 'CELERY_URL', it configures the broker.")
 
-            self.flask_app.config.update(_config)
+        if not config.get('CELERY_TASK_IGNORE_RESULT'):
+            config['result_backend']    = env.get('REDIS_URL', celery_url)
+        config['CELERY_BROKER_URL']     = env.get('REDIS_URL', celery_url)
 
-        celery_url = self.flask_app.config['CELERY_URL']
-        spool_dir  = self.flask_app.config['SPOOL_DIR']
         # Update file config with environment overrides
-        celery_url = env.get('CELERY_URL', celery_url)
+        self.flask_app.secret_key       = env.get('SECRET_KEY', config['SECRET_KEY'])
```

Comment:
See previous note about aligning spaces. This is a good example b/c it just causes code to be wrapped in my editor or on GitHub when reviewing your diff :-).

Please don't add unnecessary spaces to align operators.